protected/humhub/modules/space/widgets/MyMembership.php

```diff
@@ -34,7 +34,7 @@ public function run()
 
         return $this->render('myMembership', [
             'role' => $this->space->getUserGroup(),
-            'memberSince' => empty($membership) ? '-' : TimeAgo::widget(['timestamp' => $membership->created_at])
+            'memberSince' => empty($membership) ? '-' : empty($membership->created_at) ? '' : TimeAgo::widget(['timestamp' => $membership->created_at])
```

Comment:
How about:

```
(empty($membership) || empty($membership->created_at)) ? null : TimeAgo::widget(['timestamp' => $membership->created_at]);
```

Or is there a case we really want to display a `-` value?

Reply:
`-` value was to unify the sidebar items (role and member since) view for both members and non-members. We can also ignore displaying `Member Since` for non-members.

Reply:
I'd omit the `Member Since` for non members.